addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/DirectoryBrowsingScanRuleUnitTest.java

```diff
@@ -114,4 +116,18 @@ void shouldReturnExpectedMappings() {
                 tags.get(CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG.getTag()),
                 is(equalTo(CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG.getValue())));
     }
+
+    @Test
+    void shouldReturnExpectedExampleAlert() {
+        // Given / When
+        List<Alert> alerts = rule.getExampleAlerts();
+        // Then
+        assertThat(alerts.size(), is(equalTo(2)));
+        Alert alertOne = alerts.get(0);
```

Comment:
I guess it should be asserted the name.

Reply:
not sure what you mean with this, do you want me to test the hard coded server and evidence examples too?

Reply:
Yes but the name should be enough (the evidence is not used in the docs). Why test the confidence/risk and not the name, which is as important if not more.